packages/gatsby/cache-dir/component-renderer.js

```diff
@@ -104,7 +104,7 @@ class ComponentRenderer extends React.Component {
 
   render() {
     const pluginResponses = apiRunner(`replaceComponentRenderer`, {
-      props: this.props,
+      props: { ...this.props, pageResources: { ...this.state.pageResources } },
```

Comment:
this.state.pageResources doesn't need spread

Reply:
@KyleAMathews done and pushed